src/wolfio.c

```diff
@@ -45,6 +45,10 @@
     #include <stdlib.h>   /* strtol() */
 #endif
 
+#if defined(__ORBIS__) || defined(__PROSPERO__) /* PlayStation 4 and 5 */
+    #define gethostbyname(x) (NULL)
```

Comment:
Is this a viable workaround? Would prefer to fix the use of it below in `wolfIO_TcpConnect`. This function is used for HTTP requests for OCSP and CRL.